ui/src/views/AutogenView.vue

```diff
@@ -972,6 +972,9 @@ export default {
             break
           }
         }
+        if ((action.icon === 'delete' || ['archiveEvents', 'archiveAlerts', 'unmanageVirtualMachine'].includes(action.api)) && this.dataView) {
```

Comment:
Could this cause regression that you go back to previous screen and the action fails for some reason?

Reply:
Fixed